tests/test_fmi3_sim.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
```

Comment:
Since `test_fmi3.py` is getting large I decided to add simulation related tests here.